apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -31,29 +34,23 @@ const emptyWalletDescription = (
 );
 
 type TokensProps = {
-    allCoinTypes: string[];
-    coinBalance: bigint;
-    balances: Record<string, bigint>;
+    coinBalance: bigint | number;
+    balances: CoinBalanceType[] | [];
```

Comment:
Seems weird to have these passed down like this, why not just have this component fetch?

Reply:
done